lib/configuration.dart

```diff
@@ -66,8 +76,18 @@ class Configuration {
     this.useAdvertisingIdForDeviceId = false,
     this.useAppSetIdForDeviceId = false,
     this.appVersion,
-  }): trackingOptions = trackingOptions ?? TrackingOptions() {
+    this.deviceId,
+    CookieOptions? cookieOptions,
+    String identityStorage = '',
```

Comment:
why setting it as an empty string? Browser SDK defaults it to `'cookie'`

Reply:
I defined an enum `IdentityStorage` for possible values, and initialized it to `IdentityStorage.cookie.name` in constructor code (since you can't use extension methods in constant expressions here on line 81), though I'm realizing now that this won't show up in IDE intellisense. I'll change this to 'cookie'.